mysite/urls.py

```diff
@@ -14,8 +14,9 @@
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
 from django.contrib import admin
-from django.urls import path
+from django.urls import path,include
 
 urlpatterns = [
     path('admin/', admin.site.urls),
+    path('', include('blog.urls')),
```

Comment:
This works perfectly and is fine for this project, but you should learn about namespaces in the `include()` function. This allows each app in the project to have a seperate namespace for it's urls. 

Here you'd do somthing like: 
`path('' , include('blog.urls', namespace='blog'),` 
and in the `blog.urls` file you have to add an `app_name` label, i.e. just add the line:
 `app_name = 'blog'` before the `urlpatterns` list. 

This way in your templates you'd reference a particular url in the blog app like: 
`{% url 'blog:post_list` %}` 

By doing this we are able to have sensible url names in all our apps: 
Like you might have `blog:list` and `events:list` in some project.